Gulpfile.js

```diff
@@ -73,6 +74,12 @@ gulp.task('inject', function () {
 gulp.task('compile', function () {
     var tsProject = typescript.createProject(paths.src + '/tsconfig.json');
     return gulp.src(tsFiles)
+        .pipe(tslint({
+            formatter: 'prose'
```

Comment:
Here you go (quoted from https://www.npmjs.com/package/gulp-tslint):

> The format in which failures are outputted may be controlled by specifying a TSLint formatter. The default formatter is "prose". The available formatters include:

> "json" prints stringified JSON to console.log.
> "prose" prints short human-readable failures to console.log.
> "verbose" prints longer human-readable failures to console.log.
> "msbuild" for Visual Studio
> "vso" outputs failures in a format that can be integrated with Visual Studio Online.
> "checkstyle" for the Checkstyle development tool
> "pmd" for the PMD source code analyzer

I think prose is the default value anyway, but I wanted to include it for clarity.